index.css

```diff
@@ -78,8 +88,372 @@
     margin-left: 10%;
   }
 
+  .works-centered img {
+    width: 100%;
+  }
+
+  .attach-icon img {
+    float: left;
+    width: 5%;
+    height: 5%;
+    margin-right: 2%;
+  }
+
   .social a img {
     width: 19.99px;
     height: 19.96px;
   }
+
+  .extra {
+    margin-left: 6%;
+    margin-right: 6%;
+  }
+
+  .works-centered {
+    display: inline-flex;
+    flex-direction: column;
+    margin-top: 50%;
+  }
+
+  .works-title {
+    font-family: 'Crete Round', serif;
+    font-size: 400%;
+    margin-top: -10%;
+    text-align: center;
+  }
+
+  .works-centered hr {
+    list-style: solid;
+    width: 100%;
+    color: #212121;
+    margin-top: 4%;
+    margin-bottom: 27%;
+  }
+
+  .works-centered .multi-stories {
+    font-family: 'Crete Round', serif;
+    font-size: 350%;
```

Comment:
- [ ] Kindly use pixels instead percentages for the font size to be similar to the Figma design.